lib/cluster/membership.c

```diff
@@ -341,7 +341,7 @@ reap_crm_member(uint32_t id, const char *name)
     }
 
     search.id = id;
-    search.uname = name ? strdup(name) : NULL;
+    pcmk__copy_or_null(&search.uname, name);
```

Comment:
search isn't nulled here - right?

Reply:
Right, as I find out late yesterday evening when my tests coredumped :) Retesting the fix now ...